src/ResourceManager/Common/Commands.ResourceManager.Common/ArgumentCompleters/LocationCompleter.cs

```diff
@@ -49,12 +49,7 @@ public class LocationCompleterAttribute : ArgumentCompleterAttribute
                     {
                         try
                         {
-                            var instance = AzureSession.Instance;
-                            IResourceManagementClient client = instance.ClientFactory.CreateCustomArmClient<ResourceManagementClient>(
-                                context.Environment.GetEndpointAsUri(AzureEnvironment.Endpoint.ResourceManager),
-                                instance.AuthenticationFactory.GetServiceClientCredentials(context, AzureEnvironment.Endpoint.ResourceManager),
-                                instance.ClientFactory.GetCustomHandlers());
-                            client.SubscriptionId = context.Subscription.Id;
+                            IResourceManagementClient client = AzureSession.Instance.ClientFactory.CreateArmClient<ResourceManagementClient>(context, AzureEnvironment.Endpoint.ResourceManager);
```

Comment:
@maddieclayton we should pull in the latest from `preview` and apply these changes to the new `ResourceTypeCompleter` and `ScopeCompleter` classes. Also, it might be worth looking at whether or not we can make this change to the other places `CreateCustomArmClient` is being called where we can sue `CreateArmClient` instead.

Reply:
Done